mxd/backend-service/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# application.properties
```

Comment:
Remove line `# application.properties`

Reply:
# application.properties line is removed